crates/forge_open_router/src/anthropic/request.rs

```diff
@@ -120,13 +120,51 @@ impl TryFrom<ContextMessage> for Message {
             ContextMessage::ToolMessage(tool_result) => {
                 Message { role: Role::User, content: vec![tool_result.try_into()?] }
             }
+            ContextMessage::Attachments(attachments) => {
+                let content = attachments
+                    .into_iter()
+                    .map(Content::try_from)
+                    .filter_map(|result| result.ok())
+                    .collect::<Vec<_>>();
+
+                Message { content, role: Role::User }
+            }
         })
     }
 }
 
+impl TryFrom<Attachment> for Content {
+    type Error = ();
+
+    fn try_from(value: Attachment) -> Result<Self, Self::Error> {
+        if let ContentType::Image(ext) = value.content_type {
+            Ok(Content::Image {
+                source: ImageSource {
+                    type_: "base64".to_string(),
+                    media_type: format!("image/{}", ext),
+                    data: value.content,
+                },
+            })
+        } else {
+            Err(())
```

Comment:
We aren't handling Text files as an attachment, so we can perhaps remove it from the Context.